### ISSUE TITLE
Crash on iPad when showing activity popover

### ISSUE THREAD

**Issue by tony-hoang999** [kf99916/ProgressWebViewController#10]
It can be fixed as following
```
let activityViewController = UIActivityViewController(activityItems: [url], applicationActivities: nil)
activityViewController.popoverPresentationController?.barButtonItem = activityBarButtonItem
present(activityViewController, animated: true, completion: nil)

```
into activityDidClick()

**Comment by kf99916** [comment]
Hi @tony-hoang999,
Please help pull request to fix this issue.

**Comment by kf99916** [comment]
@tony-hoang999 , fixed with 718eb6227056ee64352ab62fc834d502275abfef

**Comment by salientsys** [comment]
Thank you @kf99916. Appreciated!
